src/main/java/LacksDataBaseProject/Exceptions/NullPointerException.java

```diff
@@ -0,0 +1,7 @@
+package LacksDataBaseProject.Exceptions;
+
+public class NullPointerException extends Exception {
```

Comment:
Taki Exception już istnieje, więc lepiej korzystać z gotowego :) choć najlepiej z mojej perspektwy wogóle nie używać npe excpetion tylko je handlować po swojej stornie :)

Reply:
tu nie rozumiem tego zdania " najlepiej...  je handlować po swojej stornie :)", bo poniżej piszesz "handlowanie nulli nie jest najelpszym pomysłem na tym poziomie, już lepiej pomyśleć o wykorzystaniu NULL object pattarnu czy też Optionali".  Więc coś źle rozumiem, bo dla mnie te info. są przeciwstawne :) Ja robiłem tak, że używałem NULL object'u i jednocześnie wyrzucałem błąd, żeby został on obsłużony "wyżej", np.         
if (lackToFind == Lack.NULL_LACK) {
            throw new NullLackException("Such Lack doesn't exist in the database.");
        }
Z tego co pamiętam takie wyrzucanie błędów na tym poziomie sugerował nam Mateusz i jak szukałem info. na Stackoverflow to też tak zrozumiałem, ale może coś pokręciłem... :) Możesz dać przykład jak to zrobić lepiej? :)

Reply:
Jeśli chodzi o handlowanie to głównie chodzi mi o robienie null checków w stylu:
if (null) {
to zrób coś :)
}
albo skorzystać z optionali :) tak by metoda nie rzucała wyjątków w stylu npe :)